internal/auth0/action/resource_trigger_action.go

```diff
@@ -204,7 +204,7 @@ func deleteTriggerAction(_ context.Context, d *schema.ResourceData, m interface{
 		return diag.FromErr(err)
 	}
 
-	var updatedBindings []*management.ActionBinding
```

Comment:
In case we are deleting a 1:1 trigger-action resource and there is no other action bounded to a trigger, this was failing because we were sending triggers: null as the default value for a slice is nil.